WebDriverAgentLib/Categories/XCUIApplication+FBHelpers.h

```diff
@@ -29,7 +29,7 @@ NS_ASSUME_NONNULL_BEGIN
 /**
  Return application elements tree in form of nested dictionaries
  */
-- (NSDictionary *)fb_tree;
+- (NSDictionary *)fb_tree:(NSArray *) excludedAttributes;
```

Comment:
Please update the docstring and add array item types as well as nullability

Reply:
what are possible values for this array?

Reply:
addressed the comment.